src/components/Catalog/Catalog.tsx

```diff
@@ -0,0 +1,18 @@
+import './Catalog.scss';
+import '../../styles/fonts/fonts.scss';
+import '../../styles/utils/vars.scss';
+import '../../styles/utils/mixins.scss';
+
+import { CategoryName } from './CategoryName/CategoryName';
+import { SortBar } from './SortBar/SortBar';
+import { ProductsList } from './ProductsList/ProductsList';
+
+import phonesFromServer from '../../phonesBase.json';
+
+export const Catalog = () => (
+  <div className="catalogContent">
+    <CategoryName productsQuantity={phonesFromServer.length} />
```

Comment:
It's just a page title, no need in a separate component for it. Use `<h1>` tag for title.